Analyse_Birds.qmd

```diff
@@ -1,1397 +1,1589 @@
+-   ***Years***: from 1991 to 2011 - *qualitative explanatory variable*
+-   ***Nest Fate***: the fate of the chicks -\> Successly fledged at least one young (1) or Failed (0)- *quantitative explanatory variable*
+-   ***Initiation Date***: Date of the first laided egg - *quantitative explanatory variable*
+-   ***Number fledged***: Number of chicks that survived and fledged the nest - *quantitative explanatory variable*
+-   ***Nest End Date***: the date when the fate of it was recorded - *quantitative explanatory variable*
+
+All theses observations are presented in a table with the following collums :
+
+-   *Nest_ID* = unique nest identifier
+-   *State* = State in which nest was located (Illinois or Wisconsin)
+-   *County* = County in the state in which nest was located
+-   *Habitat* = Grassy habitat in which nest was located (pasture, cool-season grassland, warm-season grassland, or prairie)
+-   *Species* = English common name
+-   *Year* = Year nest was found
+-   *Nest_Fate* = Successly fledged at least one young (1) or Failed (0).
+-   *Initiation_Date* = Coded date when the first egg was laid. April 1 = 1; May 1 = 31; June 1 = 62; July 1 = 92; August 1 = 123; September 1 = 154
+-   *Number_Fledged* = For a successful nest, number of young in nest on nest end date; For a failed nest, Number_Fledged is 0
+-   *Nest_End_Date* = Coded date when the nest failed or was successful. April 1 = 1; May 1 = 31; June 1 = 62; July 1 = 92; August 1 = 123; September 1 = 154
+
+# What kind of questions can we rise ?
+
+Import dataset
+
+```{r,include=FALSE}
+# Datatset import under the name of "data"
+
+data <- read.table("Bird_dataset.txt", header = TRUE, stringsAsFactors = TRUE)
+```
+
+Charge the packages
+
+```{r, include=FALSE}
+
+# Function to check and install packages if not installed
+install_if_missing <- function(packages) {
+  for (package in packages) {
+    if (!require(package, character.only = TRUE)) {
+      install.packages(package, dependencies = TRUE)
+      library(package, character.only = TRUE)
+    } else {
+      library(package, character.only = TRUE)
+    }
+  }
+}
+
+# List of required packages
+required_packages <- c(
+  "knitr",      # for R Markdown chunk options
+  "ggplot2",    # for graphs
+  "corrplot",   # for correlation matrix calculations
+  "ggcorrplot",
+  "plot3D",     # for 3D plots
+  "DHARMa",     # for model diagnostics
+  "rcompanion", # for pseudo R² in models
+  "lattice",    # for multipanel graphics
+  "dplyr",      # for data manipulation
+  "FactoMineR", # for ordinations
+  "factoextra"
+)
+
+# Install and load the packages
+install_if_missing(required_packages)
+```
+
+# [1. Data exploration]{.underline}
+
+### Repartition of the data through time
+
+##### Rudy part : Statistical tests
```

Comment:
Since Rudy's part is reviewed, we can just put the title of his part and remove "Rudy part"